packages/wrangler/src/dev/start-server.ts

```diff
@@ -311,7 +386,25 @@ export async function startLocalServer(props: LocalProps) {
 		const server = new MiniflareServer();
 		server.addEventListener("reloaded", async (event) => {
 			await maybeRegisterLocalWorker(event, props.name);
-			props.onReady?.(event.url.hostname, parseInt(event.url.port));
+
+			const proxyData: ProxyData = {
+				userWorkerUrl: {
+					protocol: event.url.protocol,
+					hostname: event.url.hostname,
+					port: event.url.port,
+				},
+				userWorkerInspectorUrl: {
+					protocol: "ws:",
+					hostname: "127.0.0.1",
+					port: props.runtimeInspectorPort.toString(),
+					pathname: `/core:user:${props.name ?? DEFAULT_WORKER_NAME}`,
+				},
+
+				headers: {},
+				liveReload: props.liveReload,
+			};
+
+			props.onReady?.(event.url.hostname, parseInt(event.url.port), proxyData);
```

Comment:
TODO(fix): using UserWorker hostname and port, needs to be the ProxyWorker

Reply:
This is fine. The devEnv is in scope in the props.onReady in the layer above this. That's where the overrides happen for UserWorker.{ip,port} -> ProxyWorker.{hostname,port}